repos/system_upgrade/el7toel8/actors/opensshprotocolcheck/tests/unit_test.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+
+from leapp.exceptions import StopActorExecutionError
+from leapp.libraries.actor import library
+from leapp.models import OpenSshConfig, OpenSshPermitRootLogin, Report
+from leapp.snactor.fixture import current_actor_context
+
+
+def test_no_config(current_actor_context):
+    with pytest.raises(StopActorExecutionError):
+        library.process(iter([]))
+
+
+osprl = OpenSshPermitRootLogin(value='no')
+
+
+@pytest.mark.parametrize('protocol', ['', '7'])
```

Comment:
Just a nit that the real values that could be in the config are `1`, `2` or `1,2` so testing these would probably be cleaner, but this does its job fine.

Reply:
I see... Updated the tests, thanks.